integration_test/third_party_apps_data/simulacra.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"context"
+	"embed"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"path"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/ops-agent/apps"
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator"
+	"github.com/GoogleCloudPlatform/ops-agent/integration_test/agents"
+	"github.com/GoogleCloudPlatform/ops-agent/integration_test/logging"
+	"github.com/google/uuid"
+
+	"github.com/GoogleCloudPlatform/ops-agent/integration_test/gce"
+)
+
+//go:embed applications
+var scriptsDir embed.FS
+
+func distroFolder(platform string) (string, error) {
+	if gce.IsWindows(platform) {
+		return "windows", nil
+	}
+	firstWord := strings.Split(platform, "-")[0]
+	switch firstWord {
+	case "centos", "rhel", "rocky":
+		return "centos_rhel", nil
+	case "debian", "ubuntu":
+		return "debian_ubuntu", nil
+	case "opensuse", "sles":
+		return "sles", nil
+	}
+	return "", fmt.Errorf("distroFolder() could not find matching folder holding scripts for platform %s", platform)
+}
+
+func setupOpsAgent(ctx context.Context, vm *gce.VM, logger *logging.DirectoryLogger, configFilePath string) error {
+	data, err := ioutil.ReadFile(configFilePath)
+	if err != nil {
+		return err
+	}
+
+	config, err := confgenerator.UnmarshalYamlToUnifiedConfig(ctx, data)
+	if err != nil {
+		return err
+	}
+
+	if err := agents.SetupOpsAgent(ctx, logger.ToMainLog(), vm, config.String()); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func getAllReceivers(config *confgenerator.UnifiedConfig) (receivers []string) {
+	for logs := range config.Logging.Receivers {
+		app := config.Logging.Receivers[logs].Type()
+		receivers = append(receivers, app)
+	}
+
+	for logs := range config.Metrics.Receivers {
```

Comment:
You can use the same suggestion as above here, if not I'd suggest at least not calling it `logs`.